covalent_ui/app.py

```diff
@@ -43,6 +47,38 @@
 WEBHOOK_PATH = "/api/webhook"
 WEBAPP_PATH = "webapp/build"
 
+app_log = logger.app_log
+log_stack_info = logger.log_stack_info
+
+
+class DaskCluster(Process):
+    def __init__(self, logger: Logger):
+        super(DaskCluster, self).__init__()
+        self.logger = logger
+        self.daemon = False
+        self.name = "DaskClusterProcess"
+
+    def run(self):
+        cluster = LocalCluster()
+        scheduler_address = cluster.scheduler_address
+        # Works only for process based dask clusters (scheduler address is inproc:// ... for
+        # thread based clusters)
+        scheduler_port = int(scheduler_address.split(":")[-1])
+        self.logger.warning(f"The Dask scheduler is running on {scheduler_address}")
+        self.logger.warning(f"Dask cluster dashboard is at: {cluster.dashboard_link}")
+        set_config(
+            {
+                "dask": {
+                    "scheduler_address": scheduler_address,
+                    "scheduler_port": scheduler_port,
+                }
+            }
+        )
+
+        # Halt the process here until its terminated
+        signal.pause()
```

Comment:
I don't think we should split the address into address and port since in the case of remote clusters there will be no port, and splitting it does not give us any advantage either afaik.

Reply:
correct... I did not make any changes to the original code in `start_cluster`.. we can update it here